daemon/containerd/image_exporter.go

```diff
@@ -308,12 +308,13 @@ func (i *ImageService) LoadImage(ctx context.Context, inTar io.ReadCloser, outSt
 			logger.WithField("alreadyUnpacked", unpacked).WithError(err).Debug("unpack")
 			return nil
 		})
-		if err != nil {
-			return errors.Wrap(err, "failed to unpack loaded image")
-		}
 
 		fmt.Fprintf(progress, "%s: %s\n", loadedMsg, name)
```

Comment:
Perhaps add a comment here that it's intentional to send the event before the error (I'm pretty sure someone at some point would otherwise change it back 🙈)